_component_design/selector-strip.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Selector Strip
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  Selector Strips allow the user to make a selection from a group of options. This pattern is a more visually prominent alternative to check boxes (multi-select) or radio buttons (single-select).
+preview-image: preview-images/selector-strip.svg
+status: stable
+resource: true
+last-modified: 2017-10-29
+---
+
+{% include toc.html %}
+
+## Overview
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### When to use
+
+Use a selector strip when you have a small group of options the user must select from. Selector strips can be functionally identical to radio buttons or check boxes (single-select or multi-select), but are typically used instead of those patterns in situations where the selection effects the rest of the page, alters the data on the page, or where the designer would like to give more visual prominence to the group of options.
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practice
+
+Selector strips function best when:
+- The labels inside the strip are short (1-3 words long max) so the number of options can fit horizontally on a small screen.
+- checkboxes or radio buttons do not make sense visually.
+- the options are important to the rest of user’s workflow and deserve a more prominent visual treatment. 	
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the selector strip pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/selector-strip/select-strip-hero.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Default layout
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The default layout has two main styles: Single-Select and Multi-Select. In both layouts the unselected options have a white background with cyan text & selected options have a cyan background with white text. 
+
+### Single-Select 
```

Comment:
Split up the image for single select and multi-select into two svgs, and set them up as separate hxRow sections.

Reply:
split files, added new section. Also moved open paragraph, "The default layout has two main styles..." into a new `hxRow` as it makes more sense with the new segmentation.